src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -945,6 +945,22 @@ public string PlaceholderText
             }
         }
 
+        /// <summary>
+        /// The Selected Text property.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SelectedText
+        {
+            get
+            {
+                string temp;
+                GetProperty(TextEditor.Property.SELECTED_TEXT).Get(out temp);
+                return temp;
+            }
```

Comment:
If this property is just a getter, I think, it would be better to write it in a description for users.

Reply:
I did not notice such information in other getter properties

I will add (read-only)